src/update.sh

```diff
@@ -0,0 +1,48 @@
+# Based On The OneUpdater workflow, but heavily simplified for github releases
```

Comment:
Small nit: needs a shebang line

Reply:
No, I don't think it does. This script is directly written into the info.plist. No shebangs in alfred scripts, because the language is specified in the plist itself.